tamm/application.php

```diff
@@ -3,13 +3,16 @@
 namespace Tamm;
 
 require_once(__DIR__.'/framework/core/bootstrap.php');
+require_once(__DIR__ . '/framework/hosting/i_hosting_environment.php');
+require_once(__DIR__ . '/framework/hosting/hosting_environment.php');
```

Comment:
@sowailem still wondering why I need to use `require_once`, the naming strategy made me crazy :)